apps/core/lib/core/models/user.ex

```diff
@@ -46,9 +46,22 @@ defmodule Core.User do
     |> validate_format(:email, @valid_email_format)
     |> unique_constraint(:email)
     |> validate_length(:password, @valid_password_length)
+    |> validate_confirmation(:password, message: "does not match password!")
     |> encrypt_passwd()
   end
 
+  def update_changeset(model, params) do
```

Comment:
You don't need an `update_changeset` to update the User registration. You can use the `registration_changeset` for this. Moreover, there is a lot of duplicated code from the latter in this changeset.